modules/con-sdpolicy-security.adoc

```diff
@@ -6,7 +6,7 @@
 = Security
 
 == Authentication provider
-Authentication for the cluster is configured as part of the link:https://cloud.redhat.com/openshift[OpenShift Cluster Manager (OCM)] cluster creation process. OpenShift is not an identity provider, and all access to the cluster must be managed by the customer as part of their integrated solution. Multiple identity providers provisioned at the same time is supported. The following identity providers are supported:
+Authentication for the cluster can be configured using either the link:https://cloud.redhat.com/openshift[OpenShift Cluster Manager (OCM)] cluster creation process or using the `rosa` CLI. The `rosa` CLI allows the use of an `htpasswd`. OpenShift is not an identity provider, and all access to the cluster must be managed by the customer as part of their integrated solution. Multiple identity providers provisioned at the same time is supported. The following identity providers are supported:
```

Comment:
If {cloud-redhat-com} is available, use it.
```suggestion
Authentication for the cluster can be configured using either the link:https://cloud.redhat.com/openshift[OpenShift Cluster Manager (OCM)] cluster creation process or using the `rosa` CLI. The `rosa` CLI allows the use of an `htpasswd`. {product-title} is not an identity provider, and all access to the cluster must be managed by the customer as part of their integrated solution. Multiple identity providers provisioned at the same time is supported. The following identity providers are supported:
```

Reply:
Done in the new PR.